packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -696,41 +697,44 @@ export class TreeViewWidget extends TreeViewWelcomeWidget {
         }
     }
 
-    protected override renderTailDecorations(node: TreeViewNode, props: NodeProps): React.ReactNode {
-        return this.contextKeys.with({ view: this.id, viewItem: node.contextValue }, () => {
+    protected override renderTailDecorations(treeViewNode: TreeViewNode, props: NodeProps): React.ReactNode {
```

Comment:
It's common practice to encode the type of a variable in dynamically typed languages like Smalltalk, wher you would call a parameter `aString`. IMO, it's just redundant information in a language like Typescript. I wouldn't want to make it a pattern.

Reply:
I understand, but it genuinely got confusing when reading the code where multiple types of nodes are being handled in the same general area. I think future readers will benefit from it in this instance.